FoundationsMastery/CustomString.cs

```diff
@@ -0,0 +1,67 @@
+﻿using FoundationsMastery.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FoundationsMastery
+{
+    public class CustomString : Interfaces.ICustomString
+    {
+        
+
+        public CustomString(char[] mychars)
+        {
+            Contents = mychars;
+        }
+
+        public CustomString()
+        {}
+
+        public IEnumerable<char> Contents
+        {
+            get; set;
+        }
+
+        public int Length
+        {
+            get;
+        }
+
+        public void Clear()
+        {
+            Contents = null;
+        }
+
+        public string Concat(IEnumerable<char> rhs)
+        {
+            string myString = "the quick brown fox jumps over the lazy dog";
+            string mySecondString = "this is the second string";
+
+            Contents = String.Concat(myString, mySecondString);
```

Comment:
The method should not modify the `Contents`. Also, you're only 2 steps away from finishing this method!